core/tests/protractor_utils/general.js

```diff
@@ -41,6 +41,14 @@ var checkForConsoleErrors = function(errorsToIgnore) {
         return !(browserLog.message.includes(' Slow network is detected.'));
       });
     }
+
+    // Currently we are using CKeditor-4 and Ckeditor-5 simultaneously we are
```

Comment:
Please change this to:
```
    // Since we are in the process of upgrading CKeditor-4 to Ckeditor-5, we are observing
    // consoles errors for duplicated modules (for more details, please refer to https://github.com/ckeditor/ckeditor5/issues/1821)
    // and therefore, we need to filter such logs until we've upgraded to CKEditor-5 completely.
```

Also, I see that the upstream issue is still open (https://github.com/ckeditor/ckeditor5/issues/1821). How do you plan to fix it when the migration is complete?

Reply:
Hi @apb7, yes the issue is still opened.
They have provided a solution to but currently, we are using two different editors one with a classic build and one with an inline build. Now, in classic build all the files are used with plugins which is one of the reasons for this error. I'll try to reframe the class build editor and try to remove the extra files after the plugins are migrated or shift to a custom CK build.

Reply:
Okay, sounds good. Thanks for the explanation. Also, I think you haven't yet correctly framed the comments. Please recheck this carefully. Thanks!

Reply:
Updated PTAL Thansk!